NDTensors/src/lib/GradedAxes/src/gradedunitrange.jl

```diff
@@ -37,6 +37,16 @@ function Base.OrdinalRange{T,T}(a::GradedOneTo{<:LabelledInteger{T}}) where {T}
   return unlabel_blocks(a)
 end
 
+# == is just a range comparison that ignores labels. Need dedicated function to check equality.
+blocklabels(::AbstractUnitRange) = nothing
```

Comment:
I think it would be better to define this as:
```julia
using FillArrays: Fill
struct NoLabel end
blocklabels(r::AbstractUnitRange) = Fill(NoLabel(), blocklength(r))
```
so that the output of `blocklabels` is more consistent across labelled and non-labelled types (i.e. it is a vector of length `blocklength`). I'm also suggesting using `FillArrays.Fill` here since we know the vector is uniform and that is an efficient representation of that (the storage is `O(1)`). Finally, I'm suggesting defining and using `NoLabel()` as a singleton type instead of `nothing`/`Nothing()`, since `nothing`/`Nothing()` is very generic (there could be other meanings to `nothing`), and generally I try to limit its usage if I can make a more specific type. Mostly we've used `nothing` as a default value for keyword arguments that aren't specified, though even there we might replace that with a singleton type such as `Unspecified()`.